src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/timetofirsthearing/TimeToFirstHearingReport.java

```diff
@@ -294,40 +294,36 @@ private ReportSummary updateOpenTrack(ReportSummary reportSummary, boolean isFir
     }
 
     private void setPercent(AdhocReportType adhocReportType) {
```

Comment:
This isn't tested at all so we really should add tests, verify that they pass on the previous code and on the new code as well.
As it's a private method (being called by other private methods higher up), go to the first public method that indirectly calls this and test it.